### PR TITLE
add new collections to menu

### PR DESCRIPTION
**Pull request by lbiedinger**

### CODE REVIEW

File: lib/europeana/page.rb [hunk 1]
```diff
@@ -176,9 +176,14 @@ def menu_data
     #
     def navigation_global_primary_nav_collections_submenu_items
```
Comment (rwd) [inline lib/europeana/page.rb]: Could we DRY this up a bit with a Hash that gets iterated over like:
```ruby
{
  '1914-1918' => 'world-war-I',
  'Art' => 'art'
  # etc
}.map do |title, slug|
 link_item(title, URI.join(europeana_collections_url, "collections/#{slug}"), is_current: false)
end
```

File: lib/europeana/page.rb [hunk 2]
```diff
@@ -175,11 +175,19 @@ def menu_data
...
+        'Art' => 'art',
+        'Fashion' => 'fashion',
+        'Maps and Geography' => 'maps',
+        'Music' => 'music',
+        'Natural History' => 'natural-history',
+        'Photography' => 'photography',
+        'Sport' => 'sport'
+      }
+      collection_titles_and_slugs.map do |title, slug|
+        link_item(title, URI.join(europeana_collections_url, 'collections/', slug), is_current: false)
```
Comment (rwd) [inline lib/europeana/page.rb]: 'collections' not 'collections/'

Comment (lbiedinger) [inline lib/europeana/page.rb]: needs the slash with URI.join because it will otherwise be ignored https://apidock.com/ruby/URI/join/class

Comment (rwd) [inline lib/europeana/page.rb]: Sorry, yes I realised that and had cancelled the comment (hence approved the PR) but it got through somehow.

**Pull request merged**